static/openid-configuration.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Why does this file exist? We should be using the openid-configuration generated by solid-idp